oct/ansible/oct/callback_plugins/log_results.py

```diff
@@ -150,13 +150,19 @@ def sanitize_results(data):
     the data and format it however they want, but we don't want to store it
     twice.
     """
-    for key in data.keys():
-        if key.startswith('_ansible'):
-            del data[key]
-    if 'results' in data:
-        for result in data['results']:
-            if 'stdout' in result and 'stdout_lines' in result:
-                del result['stdout_lines']
+    if isinstance(data, list):
```

Comment:
Can we have nested `results` fields? If not, we should have a separate recursive function call for sanitizing just the `_ansible` bits -- a smell here is that the first call to `santiize_results()` will not ever be with a list, so we have two separate cases I think -- top-level where you want to get rid of `stdout_lines` and recursively where you want to get rid of keys